configure.ac

```diff
@@ -117,4 +115,31 @@ AS_IF([test "x$with_docs_only" = xyes],
     ]) # Docs only
 
 
+AC_ARG_ENABLE([debug],
+              AS_HELP_STRING([--enable-debug], [Enable extra debugging code (default is NO).]),
+              [], [enable_debug=no])
+
+if test $enable_debug = yes; then
+	AC_DEFINE([ENABLE_DEBUG], [1], [Enable debugging code])
+	CFLAGS="$CFLAGS -O0 -g3"
+	AC_DEFINE([UCS_MAX_LOG_LEVEL], [UCS_LOG_LEVEL_TRACE_POLL], [Highest log level])
```

Comment:
Should we move this to a separate file (config/ucx.m4 or config/ucs.m4 ? We should avoid mixing UCC and UCS?